folly/synchronization/Tearable.h

```diff
@@ -63,17 +70,17 @@ class Tearable {
   // anything* with the result may trigger undefined behavior unless you've
   // verified that the data you read was consistent.
   void load(T& dst) const {
-    RawWord newDst[kNumDataWords];
+    std::array<RawWord, kNumDataWords> newDst;
```

Comment:
This is an unrelated change. Let's stick to one thesis here.

Reply:
Fair enough, I just changed it to support the minimal change of removing the intermediate type in favor of calling `std::atomic_init` in a loop. I will save the `std::array` change for another PR.